web/src/pages/Route/List.tsx

```diff
@@ -542,6 +545,33 @@ const Page: React.FC = () => {
         actionRef={ref}
         rowKey="id"
         columns={columns}
+        rowSelection={rowSelection}
+        tableAlertRender={() => (
+          <Space size={24}>
+            <span>
+            {formatMessage({ id: 'page.route.chosen' })} {selectedRowKeys.length} {formatMessage({ id: 'page.route.item' })}
+            </span>
+          </Space>
+        )}
+        tableAlertOptionRender={() => {
+          return (
+            <Space size={16}>
+              <Button
+               onClick={async () => {
+               await remove(selectedRowKeys).then(() => {
+                handleTableActionSuccessResponse(
+                  `${formatMessage({ id: 'component.global.delete' })} ${formatMessage({
+                    id: 'menu.routes',
+                  })} ${formatMessage({ id: 'component.status.success' })}`,
```

Comment:
It is better not to use this method for internationalization, Chinese and English are hardly compatible at the same time

Reply:
![image](https://user-images.githubusercontent.com/72343596/183295386-383aebee-714c-47cd-a8ff-3daf99a54f75.png)
What I mean is that we shouldn't use template splicing, it leaves extra spaces in the Chinese state.
```suggestion
                  {formatMessage({ id: 'component.global.delete.routes.success' })},
```